internal/cluster/service.go

```diff
@@ -53,21 +71,7 @@ func cleanURL(url string) string {
 }
 
 func (s *clusterService) GetClusterView(ctx context.Context) (View, error) {
-	client, err := auth.NewClient(s.authURL, s.serviceToken, s.clientOptions...)
-	if err != nil {
-		return nil, err
-	}
-	client.SetJWTSigner(
-		&goaclient.JWTSigner{
-			TokenSource: &goaclient.StaticTokenSource{
-				StaticToken: &goaclient.StaticToken{
-					Value: s.serviceToken,
-					Type:  "Bearer"}}})
-
-	res, err := client.ShowClusters(ctx, authClient.ShowClustersPath())
-	if res.StatusCode != http.StatusOK {
-		err = fmt.Errorf("got status %s (%d) from %s", res.Status, res.StatusCode, s.authURL)
-	}
+	res, err := s.authClient.ShowClusters(ctx, authClient.ShowClustersPath())
```

Comment:
Just wondering, why are not using cluster client directly, as auth proxies to cluster internally. https://github.com/fabric8-services/fabric8-auth/blob/master/controller/clusters.go#L33. 

Also we have auth and cluster client clean generated dependency available, I think we should start using it.
For why you can look at https://github.com/fabric8-services/fabric8-wit/issues/2063

Reply:
OK. I think we need to create a separate issue for this for these legacy approach app (which don't use fabric8-starter).